templates/Coaster/show.html.twig

```diff
@@ -93,15 +93,23 @@
             <button type="button" class="btn btn-default btn-block btn-float btn-float-lg text-bold">
               <i class="icon-arrow-up8 text-warning-600"></i>
               {% if coaster.height is not empty %}
-                {{ coaster.height ~ ' m' }}
+                  {% if unities.isImperial(app.request.locale) %}
+                      {{ (coaster.height * 3.281)|round(0) }} ft
```

Comment:
why not a format_unit function, doing the conversion centrally, and adding the extension also?